public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -147,12 +148,19 @@ export class AppChromeService {
 
     const lastEntry = entries[0];
     const newEntry = { name: newPageNav.text, views: [], breadcrumbs, time: Date.now(), url: window.location.href };
-    const isSameUrl = lastEntry && newEntry.url === lastEntry.url;
+    const isSamePath = lastEntry && newEntry.url.split('?')[0] === lastEntry.url.split('?')[0];
 
-    // To avoid adding an entry with the same url twice, we always use the latest one
-    if (isSameUrl) {
+    // To avoid adding an entry with the same path twice, we always use the latest one
+    if (isSamePath) {
       entries[0] = newEntry;
     } else {
+      if (lastEntry && lastEntry.name === newEntry.name) {
+        duplicateUnifiedHistoryEntry({
```

Comment:
nit about the name: it's not very clear that it is a logging event, I would call it something like `logEvent... `?

Reply:
🤔 What about:
`logClickUnifiedHistoryEntryEvent`
`logDuplicateUnifiedHistoryEntryEvent`
`logUnifiedHistoryDrawerInteractionEvent`
`logUnifiedHistoryShowMoreEvent`